docs/.vuepress/components/Card.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <a v-if="isExternal && clickable" :href="getLink" class="hide-icon flex">
+    <CardInner
+      :level="level"
+      :hero="hero"
+      :header="header"
+      :leftimg="leftimg"
+      :title="title"
+      :content="content"
+      :link="link"
+      :button="button"
+    />
+  </a>
+
+  <router-link
+    v-else-if="getLink.length && !Array.isArray(this.button) && clickable"
+    :to="getLink"
+    class="hide-icon flex"
+  >
+    <CardInner
+      :level="level"
+      :hero="hero"
+      :header="header"
+      :leftimg="leftimg"
+      :title="title"
+      :content="content"
+      :link="link"
+      :button="button"
+    />
+  </router-link>
+
+  <div v-else class="flex">
+    <CardInner
+      :level="level"
+      :hero="hero"
+      :header="header"
+      :leftimg="leftimg"
+      :title="title"
+      :content="content"
+      :link="link"
+      :button="button"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Card',
+  props: {
+    level: {
+      required: false,
+      default: 3
+    },
+    hero: {
+      required: false,
+      type: Boolean
+    },
+    header: {
+      required: false,
+      type: String
+    },
+    leftimg: {
+      required: false,
+      type: Boolean
+    },
+    title: {
+      required: true,
+      type: String
+    },
+    content: {
+      required: true,
+      type: String
+    },
+    link: {
+      required: false,
+      type: Object
+    },
+    button: {
+      required: false,
+      type: [Object, Array]
+    },
+    clickable: {
+      required: false,
+      type: Boolean,
+      default: true
+    }
+  },
+  computed: {
+    isExternal() {
```

Comment:
Similar to my comments on #839 - let's use the existing `isExternal` helper function in util.js instead of this. That way, we don't need to pass a boolean around.